src/parser/parsers/heroes.py

```diff
@@ -81,23 +93,24 @@ def _parse_hero_weapon(self, hero_value):
 
         weapon_prim_id = hero_value['m_mapBoundAbilities']['ESlot_Weapon_Primary']
         weapon_prim = self.abilities_data[weapon_prim_id]['m_WeaponInfo']
+        w = weapon_prim
 
         weapon_stats = {
-            'BulletDamage': weapon_prim['m_flBulletDamage'],
-            'RoundsPerSecond': 1 / weapon_prim['m_flCycleTime'],
-            'ClipSize': weapon_prim['m_iClipSize'],
-            'ReloadTime': weapon_prim['m_reloadDuration'],
-            'ReloadMovespeed': int(weapon_prim['m_flReloadMoveSpeed']) / 10000,
-            'ReloadDelay': weapon_prim.get('m_flReloadSingleBulletsInitialDelay', 0),
-            'ReloadSingle': weapon_prim.get('m_bReloadSingleBullets', False),
-            'BulletSpeed': self._calc_bullet_velocity(weapon_prim['m_BulletSpeedCurve']['m_spline']),
-            'FalloffStartRange': weapon_prim['m_flDamageFalloffStartRange'] / ENGINE_UNITS_PER_METER,
-            'FalloffEndRange': weapon_prim['m_flDamageFalloffEndRange'] / ENGINE_UNITS_PER_METER,
-            'FalloffStartScale': weapon_prim['m_flDamageFalloffStartScale'],
-            'FalloffEndScale': weapon_prim['m_flDamageFalloffEndScale'],
-            'FalloffBias': weapon_prim['m_flDamageFalloffBias'],
-            'BulletGravityScale': weapon_prim['m_flBulletGravityScale'],
-            #'BulletRadius': weapon_prim['m_flBulletRadius'] / ENGINE_UNITS_PER_METER,
+            'BulletDamage': w['m_flBulletDamage'],
+            'RoundsPerSecond': 1 / w['m_flCycleTime'],
+            'ClipSize': w['m_iClipSize'],
+            'ReloadTime': w['m_reloadDuration'],
+            'ReloadMovespeed': int(w['m_flReloadMoveSpeed']) / 10000,
```

Comment:
oh wait, this also rounded

Reply:
in the data its string "10000", I want to confirm this later but I believe this just represents 1x multiplier

Reply:
Ah yeah it looks like it does nothing. In the case it does change, say10000 -> 14000, this would stil read out as 1, so we'd never know. So if its inconsequential could we just leave it as a float?

Reply:
oh derp, your right. just changed it from int(x)/10000 to float(x)/10000

Reply:
wait nah you're not a derp. I realise now the int is wrapping the value BEFORE it gets divided. So not really as bad as i thought. At the very least if they change it to `10000.5` it will "matter" 😂